string transcript
int32 age
float32 confidence